docs/lesson-plans/week-2.md

```diff
@@ -0,0 +1,239 @@
+</parent>
+```
+
+> Side note: Often it'll be the case that an XML element has zero children. You can still write out both the opening and closing tags with nothing between them but XML also has a shorthand that saves some time retyping the closing tag:
+> `<element></element` === `<element />`
+
+The third aspect of XML is *attributes*. Attributes live within the opening tag of XML elements. Attributes have a name and a value, both of which are just plain strings (words). The attribute value will always be wrapped in double quotes or single quotes.
+
+```xml
+<element attributeName="attributeValue" />
+```
+
+Attributes allow you to specify extra information about an element.
+
+#### How is HTML different than XML?
+
+Like we said earlier, HTML is a subset of XML. All HTML is also XML, but not all XML is HTML.
+
+HTML defines several XML elements and gives guidelines on how they can be used together. There are a ton of different HTML elements that are defined and a lot of them aren't super useful so don't bother trying to learn all of them (google is your friend though).
+
+You can also find a full list of HTML element types on [Mozilla's MDN site](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).
+
+A couple of the important HTML elements to know are:
+
+- `<div>`: Defines a block element
+- `<span>`: Defines an inline element
+- `<p>`: Stands for paragraph, use this to wrap text
+
+For each defined element, HTML also defines a ton of different attributes that you can optionally add to an element. We'll see more of these in practice, and two important ones when we talk about CSS.
+
+In addition, for a file to be considered an HTML file, there are a couple of standard patterns you need to follow. But, because we're using React for everything, React takes care of all of that for us, so we won't cover it here. See [here](http://csis.pace.edu/~wolf/HTML/htmlnotepad.htm) for details.
+
+
+## CSS
+
+CCS is the standard for adding styling to your web page. HTML can exist without CSS but CSS cannot do anything without HTML.
+
+A CSS file is just a list of rules. A rule defines some criteria for which HTML elements it applies to, and a list of properties those elements should follow.
+
+CSS, like HTML, defines 100s of different properties for elements, some of which you will use a lot, and some you will be fine never using.
+
+Here's some example CSS that'll help contextualize the parts of CSS as we talk about them.
+
+```css
+p {
+    font-size: 12px;
+    font-weight: bold;
+}
+
+.blog-post {
+    width: 100%;
+    height: 50px;
+    color: lightblue;
+}
+
+#app {
+    text-align: center;
+    height: 400px;
+}
+
+
+```
+
+#### Selectors
+
+CSS provides about a dozen ways of defining the elements you want a rule to apply to. We'll cover 3 of the most standard. You can generally get by with these for most use cases, but if a situation requires more complex selection logic you should be aware that other methods exist and you can google them.
```

Comment:
"We'll cover 3 of the most standard." was a bit confusing when I first read it (might just be me). Possibly add something like "We'll cover 3 of the most standard methods to identify elements to apply the desired CSS styling to."